agents/World.java

```diff
@@ -119,11 +131,12 @@ public Continent getContinent() {
         }
     }
 
-    public class City {
+    public class City extends Loggable {
```

Comment:
1) `City` is not really a "subclass" of `Loggable`, in most domain model, including our domain model.
2) If you do want to make sure `City` can perform the behaviours of Loggable, then you would define `City implements Loggable`, but that is also not the case in this case as Loggable is not an interface.
3) In this case, the correct model is for a `City` to own a `Logger`, like how @lolski suggests.

Reply:
I propose that we remove the logging and implement it properly in a subsequent PR